master/vol.go

```diff
@@ -888,6 +942,17 @@ func (vol *Vol) needCreateDataPartition() (ok bool, err error) {
 	return
 }
 
+func (vol *Vol) needCreateMetaPartition() (err error) {
```

Comment:
This function name is prone to misunderstanding.

Reply:
You can directly place it at the call site, there is no need for a separate function.